views/default/js/event_calendar/event_calendar.php

```diff
@@ -148,7 +148,9 @@
 							var msg = res.message;
 							if (success) {
 								var button_text = res.button_text;
+								var button_class = res.button_class;
```

Comment:
Is there a reason why you're not changing it directly on client side instead of fetching the class from the server?